docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,578 @@
+
+### Use Decorations
+
+For text decorations, the following [Tizen.NUI.BaseComponents.TextField](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextField.html) class properties are available. All properties are editable and none of them are animatable:
+
+
+| Property                           | Type        | Description                              |
+| ---------------------------------- | ----------- | ---------------------------------------- |
+| `Text`                             | String      | Specifies the text to display in UTF-8 format.     |
+| `PlaceholderText`                  | String      | Specifies the text to display when the `TextField` is empty and inactive. |
+| `PlaceholderTextFocused`           | String      | Specifies the text to display when the `TextField` is empty with key-input focus. |
+| `FontFamily`                       | String      | Specifies the requested font family.               |
+| `FontStyle`                        | PropertyMap | Specifies the requested font style.                |
+| `PointSize`                        | Float       | Specifies the size of font in points.              |
+| `MaxLength`                        | Integer     | Specifies the maximum number of characters that can be inserted. |
+| `ExceedPolicy`                     | Integer     | Specifies how the text is truncated when it does not fit. |
+| `HorizontalAlignment`              | HorizontalAlignment | Specifies the horizontal line alignment.   |
+| `VerticalAlignment`                | VerticalAlignment   | Specifies the vertical line alignment.     |
+| `TextColor`                        | Color       | Specifies the color of the text.                          |
+| `PlaceholderTextColor`             | Vector4     | Specifies the the color of the placeholder text.               |
+| `PrimaryCursorColor`               | Vector4     | Specifies the color applied to the primary cursor.  |
+| `SecondaryCursorColor`             | Vector4     | Specifies the color applied to the secondary cursor.  |
+| `EnableCursorBlink`                | Boolean     | Specifies whether to enable or disable the cursor blink.      |
+| `CursorBlinkInterval`              | Float       | Specifies the time interval in seconds between the cursor on or off states. |
+| `CursorBlinkDuration`              | Float       | Specifies the time duration in seconds after which the cursor stops blinking. |
+| `CursorWidth`                      | Integer     | Specifies the width of the cursor.                        |
+| `GrabHandleImage`                  | String      | Specifies the image to display for the grab handle. |
+| `GrabHandlePressedImage`           | String      | Specifies the image to display when the grab handle is pressed. |
+| `ScrollThreshold`                  | Float       | Specifies when the horizontal scrolling will occur, if the cursor is closer to the control border. |

```

Comment:
| `ScrollThreshold`                  | Float       | Specifies whether the horizontal scrolling will occur, if the cursor is closer to the control border. |